app/src/main/java/com/github/drawyourpath/bootcamp/database/MockDataBase.kt

```diff
@@ -0,0 +1,73 @@
+package com.github.drawyourpath.bootcamp.database
+
+import android.graphics.Color
+import android.widget.TextView
+import java.time.LocalDate
+import java.util.*
+import kotlin.collections.ArrayList
+import kotlin.collections.HashMap
+
+class MockDataBase : Database() {
+    var usersDataBase: ArrayList<String> = ArrayList<String>()
+    var usernameToPersonalInfoDataBase: HashMap<String, PersonalInfo> = HashMap()
+    var usernameToUserGoalsDataBase: HashMap<String, UserGoals> = HashMap()
+
```

Comment:
Following the other comment, printing the error message should be done by the UI.